Tribler/Main/vwxGUI/home.py

```diff
@@ -1059,8 +1059,14 @@ def SetData(self, delayedResult):
         for torrent in torrents:
             data.append((torrent.infohash, [torrent.name], torrent, ThumbnailListItemNoTorrent))
 
-        self.list.SetData(data)
-        self.list.SetupScrolling()
+        # This may throw a PyDeadObject error when a test is finishing
+        # but a delayedtask (GuiDBHandler) calls SetData with delayedResult.get().
```

Comment:
the task should be canceled instead of having this workaround, do we know which task is triggering this?